prime-dotnet-webapi/ViewModels/Health Authority Sites/_HealthAuthoritySiteMappingProfile.cs

```diff
@@ -21,6 +21,14 @@ public HealthAuthoritySiteMappingProfile()
                 .ForMember(dest => dest.CareTypeIds, opt => opt.MapFrom(src => src.CareTypes.Select(x => x.Id)))
                 .ForMember(dest => dest.PharmanetAdministratorIds, opt => opt.MapFrom(src => src.PharmanetAdministrators.Select(x => x.Id)))
                 .ForMember(dest => dest.TechnicalSupportIds, opt => opt.MapFrom(src => src.TechnicalSupports.Select(x => x.Id)));
+            CreateMap<HealthAuthoritySite, HealthAuthoritySiteSubmissionViewModel>()
+                .ForMember(dest => dest.VendorName, opt => opt.MapFrom(src => src.HealthAuthorityVendor.Vendor.Name))
+                .ForMember(dest => dest.CareType, opt => opt.MapFrom(src => src.HealthAuthorityCareType.CareType))
+                .ForMember(dest => dest.HealthAuthorityName, opt => opt.MapFrom(src => src.HealthAuthorityOrganization.Name))
+                .ForMember(dest => dest.AuthorizedUserFullName, opt => opt.MapFrom(src =>
+                    $"{src.AuthorizedUser.Party.FirstName} {src.AuthorizedUser.Party.LastName}"))
```

Comment:
@james-hollinger let me know if this is not an issue. I initially added the `AuthorizedUserViewModel` to the `HealthAuthoritySiteSubmissionViewModel` and then create the `AuthorizedUserFullName` inside `HealthAuthoritySiteSubmissionViewModel` but that was just an extra step, also the we're only using `FirstName` and `LastName` from the `AuthorizedUserViewModel` for now.